docs/README.md

```diff
@@ -26,27 +26,27 @@ cluster as an extension to Tekton Pipelines.
 Tekton Triggers consists of a controller service that runs on your Kubernetes cluster as well as the following Kubernetes Custom Resource Definitions (CRDs) that extend
 the functionality of Tekton Pipelines to support events:
 
-*  [`EventListener`](https://github.com/tektoncd/triggers/blob/main/docs/eventlisteners.md) - listens for events at a specified port on your Kubernetes cluster.
+*  [`EventListener`](https://github.com/tektoncd/triggers/blob/master/docs/eventlisteners.md) - listens for events at a specified port on your Kubernetes cluster.
```

Comment:
We should revert these. The default branch now is main.

Reply:
Updated all Triggers docs links from `master` to `main`.